### ISSUE TITLE
Release #1

### ISSUE THREAD

**Issue by gituliar** [gituliar/fuchsia#8]
In principle, current implementation is ready for release since all three key algorithms are implemented. Hovewer, to ensure that users are satisfied with Fuchsia, I propose to include implementations of Section 7 from the Lee paper. In addition, Fuchsia should be able to reduce all three examples from R.Lee. Then we can ship our first release.

**Comment by magv** [comment]
It's not ready just yet; off the top of my head:
1) README.md must be filled in.
2) "make test" must be made to work (or is it working already?).
3) 'normalize' must not be an iterator.
4) Latex documentation must be made usable.
5) Docstrings must be re-checked. For example 'export_matrix_to_file'
doesn't explain 'fmt'.
6) Public API must be listed in '__all__'.

**Comment by gituliar** [comment]
Right, we still have some things to clean-up.

However, the priority is to reduce examples from R.Lee and C.Papadopoulos. I will tell you how it works: people do care whether Fuchsia actually is usefull and works that is if it can reduce their examples. Until we send them results they do not care if we have README file or if 'fmt' option is explained. Moreover, guess what happens when they try Fuchsia which is not able to reduce their examples...

Hence, I propose to optimize Fuchsia so that it reduces mentioned examples. Send results to the authors and after that clean-up the code and release it. Meanitme, we will work on LaTeX docs and other stuff you mentioned. What do you think?

**Comment by magv** [comment]
I think all of this can be done in parallel.

I also want to remind you that you're familiar with applications that
postpone P2 issues until all P1 issues are solved: the end result is
that nothing on P2 list ever gets worked on, and it's not a pretty sight
overall.
